lib/Report/Bandwidth.php

```diff
@@ -122,21 +118,75 @@ public function getReportForm()
     /** @inheritdoc */
     public function getReportScheduleFormData(Request $request)
     {
+        $title = __('Add Report Schedule');
+
+        $data = [];
+
+        $data['formTitle'] = $title;
+        $data['reportName'] = 'bandwidth';
+
+        return [
+            'template' => 'bandwidth-schedule-form-add',
+            'data' => $data
+        ];
     }
 
     /** @inheritdoc */
     public function setReportScheduleFormData(Request $request)
     {
+        $sanitizedParams = $this->getSanitizer($request->getParams());
+
+        $filter = $sanitizedParams->getString('filter');
+        $displayId = $sanitizedParams->getInt('displayId');
+
+        $filterCriteria['displayId'] = $displayId;
+        $filterCriteria['filter'] = $filter;
+
+        $schedule = '';
+        if ($filter == 'daily') {
+            $schedule = ReportSchedule::$SCHEDULE_DAILY;
+            $filterCriteria['reportFilter'] = 'yesterday';
+
+        } else if ($filter == 'monthly') {
+            $schedule = ReportSchedule::$SCHEDULE_MONTHLY;
+            $filterCriteria['reportFilter'] = 'lastmonth';
+
+        } else if ($filter == 'yearly') {
+            $schedule = ReportSchedule::$SCHEDULE_YEARLY;
+            $filterCriteria['reportFilter'] = 'lastyear';
+        }
+
+        $filterCriteria['sendEmail'] = $sanitizedParams->getCheckbox('sendEmail');
+        $filterCriteria['nonusers'] = $sanitizedParams->getString('nonusers');
+
+        // Return
+        return [
+            'filterCriteria' => json_encode($filterCriteria),
+            'schedule' => $schedule
+        ];
     }
 
     /** @inheritdoc */
     public function generateSavedReportName($filterCriteria)
     {
+        return ucfirst($filterCriteria['filter']). ' bandwidth report';
```

Comment:
```suggestion
        return sprintf(__('%s bandwidth report', ucfirst($filterCriteria['filter']));
```

I think we should translate these report names